pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[project]
+name = "pynalyser"
+version = "0.1.0"
```

Comment:
Let's make version dynamic here and then you can implement this for setuptools in the other pr

Reply:
ok

Reply:
How is it going? ^-^

Reply:
Good 
Will try to complete by next week as I am bit busy currently.

Reply:
yep, it's fine, no pressure!